src/lucky/memoizable.cr

```diff
@@ -21,7 +21,10 @@ module Lucky::Memoizable
   # Equality (==) is used for checking on argument updates.
   macro memoize(method_def)
     {% raise "You must define a return type for memoized methods" if method_def.return_type.is_a?(Nop) %}
-    {% raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop) %}
+    {%
+    # ameba:disable Style/IsAFilter
+    raise "All arguments must have an explicit type for memoized methods" if method_def.args.any? &.is_a?(Nop)
```

Comment:
This actually can't be updated. Filed issue on Ameba: https://github.com/crystal-ameba/ameba/issues/219